parsecmd.py

```diff
@@ -235,6 +235,18 @@ def get_parser(model_names, dataset_names):
     parser.add_argument('--sparsity-perf', action='store_true', default=False,
                         help='when determining best epoch, use sparsity as primary key')
 
+    obj_detection_args = parser.add_argument_group('Object Detection Arguments')
+    obj_detection_args.add_argument('--enable-obj-detection', '--obj-detection',
+                                    dest='obj_detection', default=False,
+                                    action='store_true',
+                                    help='Enable when training an Object Detection Model')
+
+    obj_detection_args.add_argument('--obj-detection-params', dest='obj_detection_params',
```

Comment:
The `dest` is not needed here (it will convert - to _ automatically)

Reply:
Done and commit will be amended